lib/utils.js

```diff
@@ -78,16 +78,15 @@ exports.console = Object.freeze(jpmConsole);
  *
  * @return {Object}
  */
-function getManifest () {
+function getManifest (addonDir) {
```

Comment:
Can you make this more like `getManifest ({addonDir: ...})` instead? Because [this other patch](https://github.com/mozilla-jetpack/jpm/pull/394/files#diff-50e3aa130a4f97a42ee2cf111c7b1d9dR87) needs to pass in a new option to `getManifest()` as well.